src/Tizen.NUI/src/public/Layouting/LayoutTransition.cs

```diff
@@ -69,6 +69,57 @@ public enum TransitionCondition
 
         ChangeOnRemove = 16
     }
+    
+    /// <summary>
+    /// The conditions for transitions.
+    /// </summary>
+    /// <since_tizen> 6.5 </since_tizen>
```

Comment:
could you remove <since_tizen> tags?
this tag is supposed to be added when "Add ACR" is accepted.
this looks hidden-API, so I don't think versioning tag is needed.
thank you.